src/shared/components/tc-communities/communities/iot/AssetDetail/ShareSocial.jsx

```diff
@@ -0,0 +1,76 @@
+
+/* global document window */
+/* eslint-disable jsx-a11y/href-no-hash */
+
+import React from 'react';
+
+import TwitterIcon from '../../../../../../assets/images/social/icon_twitter.svg';
+import PrintIcon from '../../../../../../assets/images/social/icon_print.svg';
+import EmailIcon from '../../../../../../assets/images/social/icon_email.svg';
+import MoreIcon from '../../../../../../assets/images/social/icon_plus.svg';
+
+import './social_media.scss';
+
+export default class ShareSocial extends React.Component {
```

Comment:
We also have social share buttons in the sidebar of challenge details page. We should look to unify / reuse the social share block in future.